care/facility/models/patient.py

```diff
@@ -2,8 +2,10 @@
 import enum
 
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 from fernet_fields import EncryptedCharField, EncryptedIntegerField, EncryptedTextField
 from partial_index import PQ, PartialIndex
+from rest_framework.serializers import ValidationError
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.serializers'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5359892)